geode-core/src/main/java/org/apache/geode/distributed/internal/membership/gms/messenger/GMSQuorumChecker.java

```diff
@@ -121,7 +125,7 @@ public NetView getView() {
 
   @Override
   public Object getMembershipInfo() {
```

Comment:
Why does this return Object? It looks like that requires us later to cast it to OldMembershipInformation and then get the Channel out of it. In the old code it looks like this was always Channel. If we could change this method to return a stronger type I think that would make the code cleaner.